app/views/schedule_rules/_schedule_rule_fields.html.haml

```diff
@@ -19,15 +19,17 @@
 
 %p= t("views.schedule_rules.discount_hint")
 
-= f.association :price_group_discounts do |pgd|
-  = pgd.label pgd.object.price_group.name
-  = pgd.number_field :discount_percent, size: 4
-  = pgd.hidden_field :price_group_id, value: pgd.object.price_group.id
-  = "%"
-  - if pgd.object.price_group == PriceGroup.base 
-    = pgd.hint :base_discount_hint
-  - elsif pgd.object.price_group == PriceGroup.external
-    = pgd.hint :external_discount_hint
+- unless @highlighted_price_group_discounts&.empty?
+  .well
+    %h4= t("views.schedule_rules.highlighted_heading")
+    = f.association :price_group_discounts, collection: @highlighted_price_group_discounts do |pgd|
+      = render partial: "price_group_discount", locals: { pgd: pgd }
+
+- unless @non_highlighted_price_group_discounts&.empty?
+  .well
+    %h4= t("views.schedule_rules.non_highlighted_heading")
```

Comment:
Maybe we should only show the label that says “Other price groups” if highlighted price groups exist.  This makes it easy to keep the existing look if that’s what folks want.  Thoughts?